<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>249</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="addDiceLayout">
     <item>
      <widget class="QPushButton" name="d4Button">
       <property name="text">
        <string>d4</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="d6Button">
       <property name="text">
        <string>d6</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="d8Button">
       <property name="text">
        <string>d8</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="d20Button">
       <property name="text">
        <string>d20</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>221</width>
      <height>531</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="playerNameLayout">
     <item>
      <widget class="QPushButton" name="name1Button">
       <property name="text">
        <string>namn 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="name2Button">
       <property name="text">
        <string>namn 2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="name3Button">
       <property name="text">
        <string>namn 3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="name4Button">
       <property name="text">
        <string>namn 4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="name5Button">
       <property name="text">
        <string>namn 5</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="name6Button">
       <property name="text">
        <string>namn 6</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="name7Button">
       <property name="text">
        <string>namn 7</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="name8Button">
       <property name="text">
        <string>namn 8</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="name9Button">
       <property name="text">
        <string>namn 9</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="name10Button">
       <property name="text">
        <string>namn 10</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>228</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="addRemovePlayerLayout">
     <item>
      <widget class="QPushButton" name="addPlayerButton">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removePlayerButton">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>242</x>
      <y>60</y>
      <width>251</width>
      <height>531</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="modifyDiceLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="amountofDiceLabel">
       <property name="text">
        <string>How many dice would you like to roll?</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="resultLabel">
       <property name="text">
        <string>Throw your dice!</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QSpinBox" name="bonusesButton"/>
     </item>
     <item row="6" column="0">
      <widget class="QPushButton" name="rollDiceButton">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QSpinBox" name="addDiesButton"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="bonusesLabel">
       <property name="text">
        <string>Do you have any Bonuses?</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>495</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuHelp"/>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
